### PR TITLE
Resolves #1188 and #794

### LINKED ISSUE #1188: Fox Taur Armor - Kateryna Petrovitch - Samanthafyre (Fixed)
Name of Item:Kat's Fox Taur Armor.
     
Item description:A set of security armor, light weight and easy to run in for a Taur, this item protects the entire body. (try to keep it short and sweet)

Sprites (or link to sprites): Its all implemented under "Wolf taur armor"

Any additional resources: Its all in the server already.

BYOND Key (Username):samanthafyre

Name of Character:Kateryna Petrovitch 

Main Job(s) of Character:Security

Why your character should have this item: Normal armor issued to the security forces of the station are designed for bi-pedals. Not taurs so the armor does not protect the rest of her body, she figured this would be a welcome change to apply for specialized armor after an incident with her being shot in the back.

Additional Notes (If it's a permit, insert the information for the permit here): Just a simple rename of a already implemented item. And somewhat a description add.

Fluff Item slots already used:0

### PR DESCRIPTION
**Pull request by JoanRisu**
Resolves #1188 and #794.

Taur armorrrrrrrr

### CODE REVIEW

File: code/modules/clothing/suits/armor_vr.dm [hunk 1]
```diff
@@ -33,6 +33,48 @@
 				H << "<span class='warning'>You need to have a wolf-taur half to wear this.</span>"
 				return 0
 
+	kate
```
Comment (SpadesNeil) [inline code/modules/clothing/suits/armor_vr.dm]: what is this doing here

File: code/modules/vore/fluffstuff/custom_clothes_vr.dm [hunk 2]
```diff
@@ -746,6 +746,22 @@
 			else
 				return 1
 
+	kate
```
Comment (SpadesNeil) [inline code/modules/vore/fluffstuff/custom_clothes_vr.dm]: also this

**Comment by SilverTalismen** [comment]
Guess you could call it Tarmor?

File: code/modules/clothing/suits/armor_vr.dm (hunk 1 above)
Comment (Cameron653) [inline code/modules/clothing/suits/armor_vr.dm]: If I'm correct, this is just a fancy way to say that it's a child of "/obj/item/clothing/suit/armor/vest/wolftaur" without making it "/obj/item/clothing/suit/armor/vest/wolftaur/kate"

Comment (SpadesNeil) [inline code/modules/clothing/suits/armor_vr.dm]: I'd rather this be consistent. I'd also prefer this be correctly added to the fluff item files with the appropriate comments attached.

**Comment by JoanRisu** [comment]
I'm not sure why custom taur armor was put in armor_vr in the first place. I just put it there to stay organized.

Also, it isn't all that inconsistent to shorthand the paths when some of our other custom items do the same. I'll move all the custom stuff into the custom files including serd's stuff but I'm not sure how the shorthand is a new thing here.

**Comment by Arokha** [comment]
My comments were about the file location not the paths. You won't be able to do relative paths in the custom item file anyway since that's not defined there.

**Comment by JoanRisu** [comment]
That's what I meant in the first statement. There was some custom taur armor code in there _before_ I added some more. That was my impression, at least.

The path comment was in regards to the inconsistency confusion that seems to be going on.

File: code/modules/vore/fluffstuff/custom_clothes_vr.dm [hunk 3]
```diff
@@ -746,6 +746,24 @@
 			else
 				return 1
 
+//samanthafyre:Kateryna Petrovitch
+//Variant
+	kate
```
Comment (Arokha) [inline code/modules/vore/fluffstuff/custom_clothes_vr.dm]: `/obj/item/clothing/head/helmet/space/fluff/joan/kate` ... ?

**Comment by Arokha** [comment]
@JoanRisu merge conflicts

**Pull request merged**